### PR TITLE
fix userform lastname can not get focus and zh-CN CONSENSUS trans

### PR DESCRIPTION
**Pull request by potterhe**

### CODE REVIEW

File: js/apps/admin-ui/src/user/UserForm.tsx [hunk 1]
```diff
@@ -282,7 +282,7 @@ export const UserForm = ({
           >
             <KeycloakTextInput
               data-testid="lastName-input"
-              id="kc-lastname"
+              id="kc-lastName"
```
Comment (jonkoops) [inline js/apps/admin-ui/src/user/UserForm.tsx:285]: Why was this `id` changed? That seems completely unrelated to the translation fix.

File: js/apps/admin-ui/public/locales/zh-CN/translation.json [hunk 2]
```diff
@@ -355,7 +355,7 @@
   "decisionStrategies": {
     "UNANIMOUS": "无异议",
     "AFFIRMATIVE": "肯定的",
-    "CONSENSUS": "一致同意"
+    "CONSENSUS": "共识"
```
Comment (jonkoops) [inline js/apps/admin-ui/public/locales/zh-CN/translation.json:358]: @xianli123 can I ask you to review this PR as a native speaker?

File: js/apps/admin-ui/src/user/UserForm.tsx (hunk 1 above)
Comment (potterhe) [reply js/apps/admin-ui/src/user/UserForm.tsx:285]: This is to fix the problem that the lastname input input box cannot get focus correctly. You can see lines 278-289. Only the "n" in this one is in lowercase, and the others are all "N". I refer to its peer "firstName", on lines 265-277.

File: js/apps/admin-ui/public/locales/zh-CN/translation.json (hunk 2 above)
Comment (potterhe) [reply js/apps/admin-ui/public/locales/zh-CN/translation.json:358]: You can refer to lines 299 and 297. Line 358 translates "CONSENSUS" into the semantics of "UNANIMOUS"

File: js/apps/admin-ui/src/user/UserForm.tsx (hunk 1 above)
Comment (jonkoops) [reply js/apps/admin-ui/src/user/UserForm.tsx:285]: Ahh yes, you are indeed correct. Nice catch!

Comment (jonkoops) [reply js/apps/admin-ui/src/user/UserForm.tsx:285]: @potterhe could you create a separate PR for this fix? We can backport this to a patch release more easily that way, as the translation mechanism has changed since we released 22.

Comment (jonkoops) [reply js/apps/admin-ui/src/user/UserForm.tsx:285]: Perfect thanks!

File: js/apps/admin-ui/public/locales/zh-CN/translation.json (hunk 2 above)
Comment (xianli123) [reply js/apps/admin-ui/public/locales/zh-CN/translation.json:358]: @jonkoops In the Chinese context, “一致同意” means that everyone agrees on something. “共识” means that everyone has the same idea. There are some slight differences between them. I agree with @potterhe 's suggestion. That makes more sense. Thanks!

Comment (jonkoops) [reply js/apps/admin-ui/public/locales/zh-CN/translation.json:358]: Perfect, then let's consider this resolved 👍

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
I've set this PR to merge automatically once the status checks pass. Thanks for your contribution @potterhe!
